Justfile

```diff
@@ -4,7 +4,13 @@ default:
 
 # build the cohort-extractor docker image
 build-cohort-extractor:
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    CI="${CI:-0}"
+    [ "$CI" == "1" ] && echo "::group::Build cohort-extractor (click to view)" || echo "Build cohort-extractor"
```

Comment:
Github [sets](https://docs.github.com/en/actions/reference/environment-variables) `CI="true"`, so if you check for that you won't need to set it manually in the workflow yaml which would be a bit cleaner.

Or you could just test that it's non-empty rather than looking for a specific value.

Reply:
:+1: didn't know Github set it already